legal-api/src/legal_api/models/review_result.py

```diff
@@ -69,3 +69,14 @@ def get_last_review_result(cls, review_id) -> ReviewResult:
                              order_by(ReviewResult.creation_date.desc()).
                              first())
         return review_result
+
+    @property
+    def json(self) -> dict:
+        """Return ReviewResult as a JSON object."""
+        return {
+            'status': self.status.name,
+            'comments': self.comments,
+            'reviewer': self.reviewer,
```

Comment:
reviewer is a user model https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/user.py#L69